docs/2-fork-accountability.md

```diff
@@ -0,0 +1,161 @@
+# Fork accountability
+
+As we know, the agreement property of the Tendermint consensus algorithm cannot be violated if there are at most *f* faulty processes in the system.
+What happen if more than *f* processes are faulty?
+
+A well-designed consensus protocol should provide some guarantees when this limit is exceeded. The most important guarantee is **fork accountability**, where the processes that caused the consensus to fail can be identified and punished according to the protocol specifications. 
+
+## The problem
+
```

Comment:
We should "formally" define the problem here.

Reply:
I changed a bit the structure of this section, now the problem is better defined. What do you mean by "formally"? Is the new version fine?